packages/dart/sshnoports/lib/src/noports_cli/util/usage_messages.dart

```diff
@@ -0,0 +1,36 @@
+class UsageMessages {
+  static final mainMenu = '''
+  
+Usage: noports <command> <argument_string>
```

Comment:
`<argument_string>` suggests there is only one argument string. But it's actually going to be `noports <command> [arg1, arg2, ...]>`

Reply:
I suggest using `brew --help` / `brew commands` / `brew help [COMMAND]` as an example for a good experience for a router/wrapper cli app